app/views/registers/show.html.haml

```diff
@@ -71,62 +70,35 @@
                         %p This register currently has no data. The owner of the register will publish the data once it has been gathered.
                 - else
                   = render partial: 'record', collection: @records
-          .pager
-            .pager-controls.js-hidden
-              = paginate @records, outer_window: 0, window: 3
-            - unless @records.last_page?
-              %div
-                = link_to_next_page @records, 'Load more', id: 'load-more-records', class: 'js-load-more', remote: true, 'aria-controls': 'records-count'
 
-      .govuk-grid-column-full.subsection__content.js-accordion-subsection#about-the-data__4
-        - unless @records.any? || @register.is_empty?
-          - if @search_term.present?
-            %script
-              ga('send', 'event', 'Search', 'No results', '#{@search_term}', {nonInteraction: true});
-          - else
-            %script
-              ga('send', 'event', 'Search', 'No results', 'Search term not present', {nonInteraction: true});
-    %hr{class: 'govuk-!-margin-top-0'}
-%dialog#modal_dialog.modal-dialog.dialog{role: "dialog", "aria-hidden" => "true", "aria-labelledby" => "dialog-title"}
-  %button{type: 'button', class: 'modal-dialog__close', "aria-label" => "Close dialog"} Close
-  .modal-dialog__inner
+          - if @show_load_more == true
```

Comment:
I *think* the `== true` is redundant here as `if @show_load_more` would resolve to the same value.